pkg/registry/common/swapip/nse_registry_test.go

```diff
@@ -46,6 +46,10 @@ func TestSwapIPNSERegistryServer_Register(t *testing.T) {
 		}),
 	)
 
+	ipMapCh <- map[string]string{
```

Comment:
Probably it would be better to do this in a loop?

Reply:
thanks fixed with variable.